docs/fx-core/spfx-help.md

```diff
@@ -1,72 +1,99 @@
+Teams Toolkit will use them for scaffolding, there's no further action that needs to be taken now. You can now retry creating a new SPFx Teams app.
 
-### If no Yeoman CLI is installed in your system
+#### If no Yeoman CLI is installed in your system
 
-1. Run this any place in a terminal: 
+1. Run this any place in a terminal:
 
 ```sh
 npm install --global yo
 ```
 
-2. Install the SPFx generator version that Teams Toolkit supports, say `1.14`: 
+2. Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
 
 ```sh
 npm install @microsoft/generator-sharepoint@1.14 -g
 ```
 
- 
-### If you have Yeoman CLI installed but it's not the correct version
+#### If you have Yeoman CLI installed but it's not the correct version
 
 Install the Yeoman CLI version that Teams Toolkit supports, say `4.3.0`:
 
 ```sh
 npm install --global yo@4.3.0
 ```
 
-### If you have Yeoman CLI installed but no SPFx generator
+#### If you have Yeoman CLI installed but no SPFx generator
 
-Install the SPFx generator version that Teams Toolkit supports, say `1.14`: 
+Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
 
 ```sh
 npm install @microsoft/generator-sharepoint@1.14 -g
 ```
 
-### If you have SPFx generator installed but it's not the correct version
+#### If you have SPFx generator installed but it's not the correct version
 
 1. If the global version is higher than the supported version
 
 You can continue with your currently installed version but please note that some of the latest features might not be supported in Teams Toolkit.
 
 2. If the global version is lower than supported
-Install the SPFx generator version that Teams Toolkit supports, say `1.14`: 
+Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
 
 ```sh
 npm install @microsoft/generator-sharepoint@1.14 -g
-```
\ No newline at end of file
+```
+
+## Yeoman scaffold
+
+By default,  locally installed `yo` is engaged to execute the following command with related _args_:
+
+```sh
+yo @microsoft/sharepoint 
+```
+
+SPFx scaffolding could fail due to unsuccessful execution of above command.
+
+#### _"Failed to run command: " yo @microsoft/sharepoint ..." , 'yo' is not recognized as an internal or external command"_
+
+If your os system is Windows,
+
+1. Check your npm and Node.js version. [Supported Node.js versions for SPFx v1.14.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-development-environment#install-nodejs) are:
+
+- Node.js v12.13.0+ (_Erbium_)
+- Node.js v14.15.0+ (_Fermium_)
+Corresponding npm version is v6.x. Please make sure you have the right version installed for both npm and Node.js.
+
+2. Check executable files for `yo` exist after package installation at the correct path under _HOME\\.fx\bin\spfx_
+
+If your os system is MacOS or Linux,
+
+1. Check executable files for `yo` exist after package installation at the correct path under _HOME\\.fx\bin\spfx\node_modules\\.bin_
```

Comment:
`_HOME/.fx/bin/spfx/node_modules/.bin_`?

Reply:
The second slash is for escape '.'
![image](https://user-images.githubusercontent.com/73154171/163745646-a35fdc33-d33a-45c5-a671-cae34fb32058.png)

Reply:
I meant for MacOS or Linux, shouldn't it be forward slash `/`?

Reply:
![image](https://user-images.githubusercontent.com/73154171/163750461-53aef29d-34d3-4ebb-b82b-a56e7f91ea5d.png)